subworkflows/local/bam_variant_calling_somatic_all/main.nf

```diff
@@ -60,7 +60,7 @@ workflow BAM_VARIANT_CALLING_SOMATIC_ALL {
             allele_files,
             loci_files,
             (wes ? intervals_bed_combined : []), // No intervals needed if not WES
-            fasta,
+            fasta.map{ meta, fasta -> [ fasta ] },
```

Comment:
Not super important right now, but shouldn't this be fixed in `BAM_VARIANT_CALLING_SOMATIC_ASCAT` instead?

Reply:
I'd rather we fix that in there actually